cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -9,23 +9,30 @@ metadata:
   name: rook-ceph
   namespace: rook-ceph
 spec:
-  # The path on the host where configuration files will be persisted. If not specified, a kubernetes emptyDir will be created (not recommended).
-  # Important: if you reinstall the cluster, make sure you delete this directory from each host or else the mons will fail to start on the new cluster.
-  # In Minikube, the '/data' directory is configured to persist across reboots. Use "/data/rook" in Minikube environment.
```

Comment:
why change the spacing on these sentences? Seems easier to read in a standard editor with one sentence per line

Reply:
I thought breaking it at around 80 chars makes it look more readable on GitHub :)
I'll revert it.

Reply:
Some people have religious wars about these sorts of things. :) IMO shorter lines doesn't make it easier to read though.

Reply:
Reverted.